release/staging-build.yml

```diff
@@ -144,6 +144,12 @@ batch:
       variables:
        PROJECT_PATH: projects/tinkerbell/tink
        CLONE_URL: https://git-codecommit.us-west-2.amazonaws.com/v1/repos/tinkerbell.tink
+  - identifier: cloudmonkey
```

Comment:
I know we arent building the other cloudstack stuff here yet, but i think we need this one since its a dep of the cli-tools image. cc @abhay-krishna

Reply:
I agree. I'm good with this change